### PR TITLE
Add missing OIDC migration

### PR DESCRIPTION
**Pull request by sergei-maertens**

### CODE REVIEW

File: src/nrc/accounts/apps.py [hunk 1]
```diff
@@ -1,5 +1,29 @@
-from django.apps import AppConfig
+from io import StringIO
+
+from django.apps import AppConfig, apps
+from django.contrib.contenttypes.management import create_contenttypes
+from django.core.management import call_command
+from django.db.models.signals import post_migrate
+
+
+def update_admin_index(sender, **kwargs):
```
Comment (stevenbal) [inline src/nrc/accounts/apps.py:9]: Is this to delete the old AppGroups from django-auth-adfs-db?

**Review by stevenbal — APPROVED** [review]
CI seems to be failing due to a (possibly unrelated) npm package.json issue? :thinking: 

`npm ci` seems to work fine locally (node v16.14.0 and npm 8.3.1)

**Comment by sergei-maertens** [comment]
ah yeah, latest npm version requires the `--legacy-peer-deps` flag, that's probably it.

File: src/nrc/accounts/apps.py (hunk 1 above)
Comment (sergei-maertens) [reply src/nrc/accounts/apps.py:9]: yes and no - we had a datamigration loading the fixture `default_admin_index.json`, but that only happens once. In new versions of ON, that fixture needs to be loaded manually. By hooking into the post-migrate signal, the fixture is reloaded on every deploy, so we can control the menu structure properly from the code.

**Pull request merged**